tikzplotlib/_patch.py

```diff
@@ -67,31 +67,27 @@ def draw_patchcollection(data, obj):
     """Returns PGFPlots code for a number of patch objects.
     """
     content = []
-    # Gather the draw options.
-    try:
-        ec = obj.get_edgecolor()[0]
-    except IndexError:
-        ec = None
-
-    try:
-        fc = obj.get_facecolor()[0]
-    except IndexError:
-        fc = None
-
-    try:
-        ls = obj.get_linestyle()[0]
-    except IndexError:
-        ls = None
 
-    try:
-        w = obj.get_linewidth()[0]
-    except IndexError:
-        w = None
+    def _or_None(x):
+        if len(x) == 0:
```

Comment:
how about `return [None] if len(x) == 0 else x`?

Reply:
Done

Reply:
Forgot to push?

Reply:
Pushed to the wrong branch, whoops